.github/workflows/docker-build-and-push.yaml

```
@@ -0,0 +1,35 @@
+name: Build and Push Docker Image
+
+on:
+  push:
+    branches:
+      - add-CDCI-artifact-registry  # Change this to your target branch
```

should this be main?
```suggestion
on:
  push:
    branches:
      - main
```

---

yes you're right this was for testing on branch